arch/arm/core/aarch64/Kconfig

```diff
@@ -90,6 +90,83 @@ config GEN_ISR_TABLES
 config GEN_IRQ_VECTOR_TABLE
 	default n
 
+config ARM_MMU
+	bool "ARM MMU Support"
+	help
+	  Memory Management Unit support.
+
+if ARM_MMU
+
+config MAX_XLAT_TABLES
+	int "Maximum numbers of translation tables"
```

Comment:
Do we need to add a range property to the Kconfig symbol?

Reply:
I don't think so. User can specify number of numbers of translation tables per requirement.